package.json

```diff
@@ -148,9 +148,7 @@
     "filehound": "^1.17.4",
     "fs-extra": "^8.0.1",
     "gulp": "^4.0.1",
-    "gulp-coffee": "^2.2.0",
     "gulp-inline-source": "^2.1.0",
-    "gulp-shell": "^0.5.2",
```

Comment:
When I converted to js typescript told me that we didn't actually use this anyway